packages/devops/scripts/ci/deployBackend.sh

```diff
@@ -7,7 +7,15 @@ DEPLOYMENT_SSH_URL=$(${DIR}/determineDeploymentSshUrl.sh)
 DEPLOYMENT_URL=$DEPLOYMENT_SSH_URL # ssh url will resolve to tupaia web frontend desktop over HTTP
 if curl --output /dev/null --silent --head --fail $DEPLOYMENT_URL; then
     echo "Deployment for ${CI_BRANCH} exists, updating with latest changes"
-    ssh -o ServerAliveInterval=15 ubuntu@$DEPLOYMENT_SSH_URL "cd tupaia/packages/${PACKAGE} && yarn build && pm2 delete ${PACKAGE} || true && pm2 start --name ${PACKAGE} dist --wait-ready --listen-timeout 15000 --time;"
+
+    REPLICATION_PM2_CONFIG=''
+    if [[ $PACKAGE == "web-config-server" ]]; then
+        # as many replicas as cpu cores - 1
+        REPLICATION_PM2_CONFIG='-i -1'
```

Comment:
N-1 leaves 1 core for kernel/other processes. If web-config-server gets overloaded other processes can still run.